### ISSUE TITLE
0.15.0 release

### ISSUE THREAD

**Issue by adamralph** [config-r/config-r#239]
**Ready** when all other issues forming part of the release are **Done**.

- [ ] close all issues on this milestone
- [ ] ensure patches have been merged from master to dev
- [ ] run code analysis in VS in *Release* mode and address violations (send a regular PR which must be merged before continuing)
- [ ] create PR from upstream dev to master
- [ ] check build, merge PR
- [ ] check build, create release in GitHub UI **on master** including releaseNotes, mentioning non-owner contributors, if any
- [ ] push nuget package
- [ ] copy release notes from GitHub to NuGet
- [ ] de-list pre-release nuget packages if present
- [ ] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [ ] post tweet in https://gitter.im/config-r/config-r
- [ ] post links to the NuGet release, GitHub release, and Tweet in each issue in this milestone, with thanks to contributors
- [ ] increment minor version on branch from dev
- [ ] push to origin branch, create PR to upstream dev
- [ ] check build, merge PR
- [ ] create a new milestone for the next release
- [ ] create new issue based on this one for next release, adding it to the new milestone
- [ ] update samples/raise issue to do so
- [ ] close this milestone

**Comment by adamralph** [comment]
This could, potentially, be 1.0, given it contains https://github.com/config-r/config-r/issues/241
